### ISSUE TITLE
Validate POMs during regular CI

### ISSUE THREAD

**Issue by lenaschoenburg** [camunda/camunda#9694]
**Description**

We've had a few releases now that failed because some POMs were not valid. Most recently here: https://github.com/camunda/zeebe/pull/9693

It'd be nice to validate the POMs during CI. Hopefully that is something that is possible via spotless or some other maven plugin.

**Comment by menski** [comment]
We are not sure why this was not catched earlier, and isn't validating the pom part of the normal maven tooling? Is this something which was rejected by the sonatype nexus during upload?

**Comment by oleschoenburg** [comment]
> Is this something which was rejected by the sonatype nexus during upload

Yes, the POM is technically valid but not complete enough for maven central, for example because they lack a `<name>` or `<packaging>`.

**Comment by Zelldon** [comment]
We have no regular dry releases, which could also catch this

**Comment by liliancavalet** [comment]
We have this task https://github.com/camunda/camunda/issues/40004, that should also fix this issue.
